### ISSUE TITLE
ez_setup.py: No such file or directory

### ISSUE THREAD

**Issue by bennypi** [EMCECS/ECS-CommunityEdition#196]
When running step1_ecs_singlenode_install.py, the following error occurs:
```
[29/Mar/2017 11:09:21] INFO [root:463] Backup object properties files
[29/Mar/2017 11:09:21] INFO [root:467] Backup application config file
[29/Mar/2017 11:09:22] INFO [root:471] Backup common-object properties file
[29/Mar/2017 11:09:22] INFO [root:475] Backup ssm properties file
[29/Mar/2017 11:09:22] INFO [root:479] Copy object properties files to host
[29/Mar/2017 11:09:22] INFO [root:483] Copy common-object properties files to host
[29/Mar/2017 11:09:22] INFO [root:487] Copy ssm properties files to host
[29/Mar/2017 11:09:22] INFO [root:491] Modify BlobSvc config for single node
[29/Mar/2017 11:09:22] INFO [root:495] Modify Directory Table config for single node
[29/Mar/2017 11:09:22] INFO [root:499] Modify SSM config for small footprint
[29/Mar/2017 11:09:22] INFO [root:503] Copy modified files to container
[29/Mar/2017 11:09:23] INFO [root:515] Adding python setuptools to container
python: can't open file 'ez_setup.py': [Errno 2] No such file or directory
[29/Mar/2017 11:09:23] INFO [root:518] Adding python requests library to container
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   136    0   136    0     0    306      0 --:--:-- --:--:-- --:--:--   306
  0     0    0 3424k    0     0  1272k      0 --:--:--  0:00:02 --:--:-- 2261k

<Output from tar removed: kennethreitz-requests-993cbf5... >

Traceback (most recent call last):
  File "setup.py", line 9, in <module>
    from setuptools import setup
ImportError: No module named setuptools
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 12537  100 12537    0     0    97k      0 --:--:-- --:--:-- --:--:--   98k
[29/Mar/2017 11:09:26] INFO [root:524] Cleaning up python packages
[29/Mar/2017 11:09:27] INFO [root:527] Flush VNeST data
[29/Mar/2017 11:09:31] INFO [root:530] Stop container
ecsstandalone
[29/Mar/2017 11:09:42] INFO [root:533] Start container
ecsstandalone
```

This can be fixed if the line 523 in step1_ecs_singlenode_install.py is moved to line 516 so that the file is retrieved before it gets executed.

**Comment by imYerik** [comment]
You need install python setup.py module first.

**Comment by padthaitofuhot** [comment]
This should be resolved in newest ECS 3.0.0 image emccorp/ecs-software-3.0.0; setup.py should be installed by default now.  Let me know if you're still seeing this issue.
